configs/swin_transformer/README.md

```diff
@@ -48,6 +62,61 @@ The pre-trained models on ImageNet-21k are used to fine-tune, and therefore don'
 | :----: | :---------------------------------------------------------: | :--------: | :-------: | :------: | :-------: | :--------------------------------------: | :---------------------------------------------------------: |
 | Swin-L | [ImageNet-21k](https://download.openmmlab.com/mmclassification/v0/swin-transformer/convert/swin-base_3rdparty_in21k-384px.pth) |  384x384   |  195.51   |  100.04  |   91.87   | [config](./swin-large_8xb8_cub_384px.py) | [model](https://download.openmmlab.com/mmclassification/v0/swin-transformer/swin-large_8xb8_cub_384px_20220307-1bbaee6a.pth) \| [log](https://download.openmmlab.com/mmclassification/v0/swin-transformer/swin-large_8xb8_cub_384px_20220307-1bbaee6a.log.json) |
 
+## How to use it?
+
+<!-- [TABS-BEGIN] -->
+
+**Predict image**
+
+```python
+>>> import torch
+>>> from mmcls.apis import init_model, inference_model
+>>>
+>>> model = init_model('configs/swin_transformer/swin-base_16xb64_in1k.py', 'https://download.openmmlab.com/mmclassification/v0/swin-transformer/convert/swin_tiny_patch4_window7_224-160bb0a5.pth')
```

Comment:
base config does not match the tiny checkpoint

Reply:
done